### DISCUSSION TITLE
NEXTAUTH_URL/NEXT_PUBLIC and Reverse Proxy

### DISCUSSION THREAD

**Discussion by Dvalin21** [blinkospace/blinko#310] [General]
Hello, 

  I have set Nextauth and public set to my domain behind nginx proxy manager, but it seems like it can't reach it.

I have set as follows

NEXTAUTH_URL: https://blinko.example.com
NEXT_PUBLIC: https://blinko.example.com

BTW: Since your last update, the username and password default is render useless as your able to 
sign up for the initial account. You may want to remove that from the main page so that it doesn't cause confusion.

When I get the container up and running and then trying to access via blinko.example.com, I just get a 502 error as if it can't be reached

**Comment by thinkbig1979** [comment]
Similar or same issue issue for me. (latest version @ 25 Jan 2025).

I have set Blinko up behind nginx proxy manager, like I have for many others.

![image](https://github.com/user-attachments/assets/bf33d589-8d7f-4a73-ae3f-30d945e8e4a7)

Mapping port 1111 to my host directly in docker allows me direct access to Blinko at `mydomain.example:1111`, so I know it's running fine, however nginx cannot seem to proxy it.

blinko-website is definitely in my nginx docker network, I've verified that.

I've tried commenting out the two env variables listed in the OP, and numerous other things, but no luck.

**Reply by sivert-io** [reply]
Not sure about your setup, but are you able to curl the blinko url from your NPM VM? 

I don't think this is an issue with blinko per se, the next auth url simply specifies the redirect url once you sign in, which should be the root url of your domain (if I'm not mistaken).

It's not the same, but I'm running my service through cloudflare tunnels with no issues.

**Reply by thinkbig1979** [reply]
Whatever the problem was, it seems to have resolved itself. After a few weeks of not touching this, I pulled the latest version and fired it up this afternoon. Left the config exactly as it was. Worked first time :-)

**Comment by akrauss** [comment]
Same issue here, any solutions?
